Packs/SymantecEDR/Integrations/SymantecEDR/SymantecEDR.yml

```diff
@@ -0,0 +1,1824 @@
+      description: 'The name of the feature that originated the event. Example: Search.'
+      type: String
+    - contextPath: SymantecEDR.AuditEvent.device_ip
+      description: The IPv6 or IPv4 address of the device that originated the event.
+      type: String
+    - contextPath: SymantecEDR.AuditEvent.user_name
+      description: The username or ID that originated or caused the event.
+      type: String
+    - contextPath: SymantecEDR.AuditEvent.atp_node_role
+      description: ''
+      type: Number
+    - contextPath: SymantecEDR.AuditEvent.category_id
+      description: The event type category. 4 = Audit.
+      type: Number
+    - contextPath: SymantecEDR.AuditEvent.device_uid
+      description: 'Unique ID of the device that originated the event. Example: 7c056576-860b-4eb9-b49c-3c349edb733f.'
+      type: String
+    - contextPath: SymantecEDR.AuditEvent.log_name
+      description: 'The index of the event. Note: This is for informational purpose and cannot be used as a filter. Use time as start_time to query for events. Example: epmp_events-2015-11-05.'
+      type: String
+    - contextPath: SymantecEDR.AuditEvent.count
+      description: ''
+      type: Number
+    - contextPath: SymantecEDR.AuditEvent.device_name
+      description: The device name (i.e., the name of the endpoint or appliance associated with an event).
+      type: String
+    - contextPath: SymantecEDR.AuditEvent.message
+      description: Human-readable (multi-line) event message or description of the event.
+      type: String
+    - contextPath: SymantecEDR.AuditEvent.log_time
+      description: 'The time the event was logged. Example: YYYY-MM-DDTHH:MM:SS.sssZ.'
+      type: Date
+    - contextPath: SymantecEDR.AuditEvent.severity_id
+      description: 'Event severity that specifies the importance of the event. Values are:  1 = info (default), 2 = warning, 3 = minor, 4 = major, 5 = critical, 6 = fatal.'
+      type: Number
+    - contextPath: SymantecEDR.AuditEvent.device_cap
+      description: 'Name or caption of the ATP appliance that generated the event. Example: EDR.'
+      type: String
+    - contextPath: SymantecEDR.AuditEvent.id
+      description: The event identifier. 1 = Exists, 2 = Partial.
+      type: Number
+    - contextPath: SymantecEDR.AuditEvent.device_time
+      description: 'The timestamp (in ISO 8601 format) that specifies the time at which the event occurred. Example: YYYY-MM-DDTHH:MM:SS.sssZ.'
+      type: Date
+    - contextPath: SymantecEDR.AuditEvent.product_ver
+      description: 'The version of the product that originated the event. Example: 4.6.8-8.'
+      type: String
+    - contextPath: SymantecEDR.AuditEvent.device_end_time
+      description: ''
+      type: Date
+    - contextPath: SymantecEDR.AuditEvent.type_id
+      description: 'The unique identifier for an event. The following events are supported: For type_id details refer to “Input/Output parameter notes”.'
```

Comment:
There is something missing in this description.  
-  The following events are supported: ????
- For type_id details refer to “Input/Output parameter notes” - Where are these notes?